README.md

```diff
@@ -37,15 +44,9 @@ Here is an example of the use of scores:
 array(2.)
 ```
 
-## Further Information on Scores and Metrics Included in the Package
-
-'scores' is a modular scoring package containing verification metrics, error functions, training scores and other statistical functions. It is primarily aiming to support the geoscience and earth system community. Scores is focused on supporting xarray and pandas datatypes for earth system data. It has wider potential application in machine learning and domains other than meteorology, geoscience and weather but primarily supports those fields. It aims to be compatible with geopandas, pangeo and work with NetCDF4 and hdf5 data sources among others.
-
-'scores' includes novel scores not commonly found elsewhere (e.g. FIRM and FlipFlip index), complex scores (CRPS), more common scores (MAE, RMSE) and statistical tests (such as the Diebold Mariano test). Scores provides its own implementations where relevant to avoid extensive dependencies, and its roadmap includes a comprehensive implementation of optimised, reviewed and useful set of scoring functions for verification, statistics, optimisation and machine learning.
-
-All of the scores and metrics in this package have undergone a thorough statistical and scientific review.
+## Acknowledgments
```

Comment:
Acknowledgements seems like the wrong title since we don't use anything of theirs.

Perhaps "Related packages"

Reply:
I agree, and that 'Related Packages' is reasonable wording